### ISSUE TITLE
UX Board review : Overall accessibility

### ISSUE THREAD

**Issue by romalpani** [dotnet/machinelearning-modelbuilder#903]
**System Information (please complete the following information):**
 - Model Builder Version: 16.1.1.2036504
 - Visual Studio Version: 

**Describe the bug**

- Regarding ‘Local ML’ and ‘Azure ML’ tags – Dante mentioned some accessibility guidance is coming in the near future for labels/tags. 
- We need to work on the keyboard interaction for model builder. Dante mentioned that the current state might fail lot of A11y tests.
- A11y: For the new UI for multi-selecting ‘features’, sync with Dante on A11y.

**Comment by beccamc** [comment]
- Per conversation with Dante tags are fine. Should not be tab-able. Be thoughtful about how screenreader interacts with them. 
- First pass at keyboard interaction is done. 
- Multi select features control is being removed.
